boards/omnigul/omnigul.config

```diff
@@ -0,0 +1,52 @@
+# Configuration for a Acer Chromebook Plus 515 (CB515-2H/OMNIGUL)
+CONFIG_COREBOOT_CONFIG=config/coreboot-omnigul.config
+
+export CONFIG_COREBOOT=y
+export CONFIG_COREBOOT_VERSION=mcb
+export CONFIG_LINUX_VERSION=6.5.13
+
+CONFIG_CRYPTSETUP2=y
+CONFIG_FLASHROM=y
+CONFIG_FLASHTOOLS=y
+CONFIG_GPG2=y
+CONFIG_KEXEC=y
+CONFIG_UTIL_LINUX=y
+CONFIG_LVM2=y
+CONFIG_MBEDTLS=y
+CONFIG_PCIUTILS=y
+CONFIG_POPT=y
+CONFIG_QRENCODE=y
+CONFIG_TPMTOTP=y
+CONFIG_HOTPKEY=n
+
+#Enable TPM2 pcap output under /tmp
+export CONFIG_TPM2_CAPTURE_PCAP=y
+
+#TPM2 requirements
+export CONFIG_TPM2_TOOLS=y
+export CONFIG_PRIMARY_KEY_TYPE=ecc
+CONFIG_TPM2_TSS=y
+CONFIG_OPENSSL=y
+
+# Boot Options (GUI)
+CONFIG_CAIRO=y
+CONFIG_FBWHIPTAIL=y
+export CONFIG_AUTO_BOOT_TIMEOUT=5
+export CONFIG_BOOT_DEV="/dev/sda2"
+export CONFIG_BOOT_KERNEL_ADD=""
+export CONFIG_BOOT_KERNEL_REMOVE=""
+export CONFIG_BOOTSCRIPT=/bin/gui-init
+
+# Boot Security / Flash Options
+export CONFIG_BOOT_REQ_HASH=n
+export CONFIG_BOOT_REQ_ROLLBACK=n
+export CONFIG_BOARD_NAME="Google Omnigul"
+export CONFIG_FLASHROM_OPTIONS="--noverify-all -p internal --ifd --image bios"
```

Comment:
ideally since me is packed, ifd is packed, we would not go "legacy" boards here and prepare images with all blbs required to flash the whole spi, not just the bios region. 

I haven't checked the specs of the baord yet, but iGPU.dGPU and firmwares their firmware will probably need to be added, otherwise jumping from Heads to final OS in kexec call will be problematic to say the least. 

Can you post specs on the PR as well, or add them in the board config?

Reply:
Any guidance on extracting the other firmware would be good. I have a stock Google ROM to work with that I took when I first flashed it to Coreboot before moving it to heads.

Not sure what you mean about the issues with kexec - this setup as posted works for both Ubuntu 23.10 and Qubes 4.2.

Let me know what specs you want, including lspci, dmidecode etc...